Useful.md

```
@@ -0,0 +1,383 @@
+    .option("-s, --sourcemap", "Generate sourcemap")
+    .action(({ output, minify, sourcemap }) => {
+      console.log(`Building to ${output}...`);
+      if (minify) console.log("Minifying...");
+      if (sourcemap) console.log("Generating sourcemap...");
+    })
+  .parse(Deno.args);
+```
+
+### Documentation
+
+Full documentation available at: https://cliffy.io/docs/command/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)